### PR TITLE
fix: dependabot

### PR DESCRIPTION
**Pull request by apvale**
## Description of the change

One more try on this.

## Testing the change

- [ ] Write your testing instructions here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [x] Non-Breaking Change (change to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.

### CODE REVIEW

File: .github/workflows/visual_regression_tests.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   install-cache:
     name: Install and cache dependencies
-    if: github.triggering_actor != 'dependabot[bot]' || contains(github.event.pull_request.draft, false)
+    if: ${{ github.triggering_actor != 'dependabot[bot]' || contains(github.event.pull_request.draft, false) }}
```
Comment (padulafacundo) [inline .github/workflows/visual_regression_tests.yml:15]: I don't think we need the `${{ }}`

Comment (padulafacundo) [reply .github/workflows/visual_regression_tests.yml:15]: Also, why `contains` instead of `==`?

**Review by padulafacundo — APPROVED** [review]
🤞

**Review by alinurr1 — APPROVED** [review]

File: .github/workflows/visual_regression_tests.yml (hunk 1 above)
Comment (apvale) [reply .github/workflows/visual_regression_tests.yml:15]: I tried to use it without the `${{ }}` and it didn't word. :(

Comment (apvale) [reply .github/workflows/visual_regression_tests.yml:15]: `contains` or `==` it's the same. I saw this function here https://docs.github.com/en/actions/learn-github-actions/expressions#contains and wanted to use it. 🙏🏻

**Pull request merged**